addons/website/models/mixins.py

```diff
@@ -8,6 +8,7 @@
 from odoo.http import request
 from odoo.osv import expression
 from odoo.exceptions import AccessError
+from urllib.parse import urljoin
```

Comment:
Import order, see https://github.com/odoo/odoo/pull/83508#discussion_r803814729
Also note that I was refering to werkzeug's url_join not this one :)